src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/ApplicationFrameworkValueProvider.cs

```diff
@@ -8,41 +8,27 @@ namespace Microsoft.VisualStudio.ProjectSystem.Properties
     [ExportInterceptingPropertyValueProvider(
     new[]
     {
-        ApplicationFrameworkProperty,
-        EnableVisualStylesProperty,
-        SingleInstanceProperty,
-        SaveMySettingsOnExitProperty,
-        HighDpiModeProperty,
-        AuthenticationModeProperty,
-        ShutdownModeProperty,
-        SplashScreenProperty,
-        MinimumSplashScreenDisplayTimeProperty
+        PropertyNameProvider.ApplicationFrameworkProperty,
+        PropertyNameProvider.EnableVisualStylesProperty,
+        PropertyNameProvider.SingleInstanceProperty,
+        PropertyNameProvider.SaveMySettingsOnExitProperty,
+        PropertyNameProvider.HighDpiModeProperty,
+        PropertyNameProvider.AuthenticationModeProperty,
+        PropertyNameProvider.ShutdownModeProperty,
+        PropertyNameProvider.SplashScreenProperty,
+        PropertyNameProvider.MinimumSplashScreenDisplayTimeProperty
```

Comment:
Since you moved the constants to another class, you can use a [using static](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive#static-modifier) statement at the top of these files, so you don't need to specify `PropertyNameProvider.` before all these constants.

Reply:
Oh, that's a great idea, thanks Yanni!